Bottom: 2dc4bca46d4dabc805a9574b535d3a0a13e95953
Top:    917d75188b0b01634d96bd607ea8547ceb416128
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-09-12 02:07:20 +0100

Win32: Select the correct GL libraries to link against


---

diff --git a/configure.ac b/configure.ac
index 74d6e9e..cd696a2 100644
--- a/configure.ac
+++ b/configure.ac
@@ -429,7 +429,7 @@ Either do not use --enable-gl or enable the gtk HID.])
 	esac
 
 	GLU_CFLAGS=
-	GLU_LIBS=-lGLU
+	GLU_LIBS=-lglu32 -lopengl32
 
 	AC_DEFINE([ENABLE_GL], 1,
 		[Define to 1 if GL support is to be compiled in])
